powerauth-java-cmd-lib/src/main/java/io/getlime/security/powerauth/lib/cmd/steps/v3/CreateActivationStep.java

```diff
@@ -86,6 +87,7 @@ public JSONObject execute(StepLogger stepLogger, Map<String, Object> context) th
 
         if (stepLogger != null) {
             stepLogger.writeItem(
+                    "activation-create-start",
```

Comment:
I would use `create-custom-activation` (or `activation-create-custom`)  prefix here. See my related comment in `v2/CreateActivationStep`.